src/ajax/check_hgvs_dialogue.php

```diff
@@ -572,9 +589,13 @@ function update_images_per_step($sStep, $sImage)
         oGenomicField.val("' . $sMappedGenomicVariant . '").attr("class", "accept");
         oGenomicField.siblings("img:first").attr({src: "gfx/check.png", title: "Validated"});
         ');
+
+        // Adding the variant to the array of validated variants to mark it as validated.
+        $_SESSION['VV']['validated_variants'][$sGBID] = $sMappedGenomicVariant;
     }
 
 
+    // TODO: Check whether the below code does not conflict with the $_SESSION enforce method.
```

Comment:
The code below fills in fields that were left empty after a call to VariantValidator and disables there `onChange`. This was done because the original code worked in a way where users could not fill in the empty fields without there being a call to the `check_hgvs_dialogue.php` that would then empty all fields that were just nicely filled by VariantValidator. However, because our new code does not clear out all variants, the `onChanges` may remain. However, filling in the fields with _some_ sort of input seems clean to me, so I have left that. @ifokkema, if you disagree on filling the empty fields, that is totally fine and it can be removed!

Reply:
Has been removed!